examples/cleanup-controller/README.md

```diff
@@ -0,0 +1,40 @@
+# Cleanup Controller
```

Comment:
Can you move the file here https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/tree/master/docs

Reply:
Done